src/explore-education-statistics-admin/src/pages/edit-release/components/EditReleasePageTemplate.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode } from 'react';
+import PreviousNextLinks, {
+  PreviousNextLink,
+} from '@admin/components/PreviousNextLinks';
+import NavLink from '@admin/components/NavLink';
+import Page from '../../../components/Page';
+
+interface Props {
+  releaseId: string;
+  children: ReactNode;
+  publicationTitle: string;
+  previousLink?: PreviousNextLink;
+  nextLink?: PreviousNextLink;
+}
+
+const EditReleasePageTemplate = ({
+  releaseId,
+  publicationTitle,
+  children,
+  previousLink,
+  nextLink,
+}: Props) => {
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        {
+          link: '/admin-dashboard',
+          name: 'Administrator dashboard',
+        },
+        { name: 'Edit release', link: '#' },
+      ]}
+    >
+      <h1 className="govuk-heading-l">
+        {publicationTitle}
+        <span className="govuk-caption-l">Edit release</span>
+      </h1>
+      <nav className="app-navigation govuk-!-margin-bottom-9">
+        <ul className="app-navigation__list govuk-!-margin-bottom-0">
+          <li>
+            <NavLink
```

Comment:
Would map the `editReleaseRoutes` to `NavLink`s here

Reply:
Yup good point, will do!

Reply:
Done, neat